src/Pages/category/CategoryList/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import CategoryListItem from './CategoryListItem';
+
+function CategoryList({ categoryList, onChangeClickId }) {
+  return (
+    <CategoryListContainer>
```

Comment:
```suggestion
    <ListContainer>
```
어차피 category 관련 내용인 것을 알고 있으니 이름을 줄이면 어떨까요?

Reply:
안그래도 이름이 자꾸 길어지는 것 같아서 고민이었는데 수정했습니다!